package.json

```diff
@@ -11,14 +11,13 @@
     "start-production": "grunt build && CONFIG_FILES=server/config/local.json,server/config/production.json grunt serverproc:dist",
     "start-circle": "grunt build && CONFIG_FILES=server/config/fxaci.json,server/config/production.json grunt serverproc:dist",
     "start-remote": "scripts/run_remote_dev.sh",
-    "test": "intern-runner config=tests/intern",
-    "test-circle": "intern-runner config=tests/intern_functional_circle fxaAuthRoot=https://fxaci.dev.lcip.org/auth fxaEmailRoot=http://restmail.net fxaOauthApp=https://oauth-fxaci.dev.lcip.org fxaUntrustedOauthApp=https://321done-fxaci.dev.lcip.org fxaProduction=true bailAfterFirstFailure=true",
-    "test-functional": "intern-runner config=tests/intern_functional",
-    "test-functional-oauth": "intern-runner config=tests/intern_functional_oauth",
-    "test-latest": "intern-runner config=tests/intern_functional_full fxaAuthRoot=https://latest.dev.lcip.org/auth/v1 fxaContentRoot=https://latest.dev.lcip.org/ fxaEmailRoot=http://restmail.net fxaOauthApp=https://123done-latest.dev.lcip.org/ fxaUntrustedOauthApp=https://321done-latest.dev.lcip.org/ fxaProduction=true fxaToken=https://token.dev.lcip.org/1.0/sync/1.5",
-    "test-sauce": "intern-runner config=tests/intern_sauce",
-    "test-server": "intern-client config=tests/intern_server",
-    "test-travis": "intern-runner config=tests/intern_travis"
+    "test": "node tests/intern.js --unit=true",
+    "test-circle": "node tests/intern.js --suites=circle --fxaAuthRoot=https://fxaci.dev.lcip.org/auth --fxaEmailRoot=http://restmail.net --fxaOauthApp=https://oauth-fxaci.dev.lcip.org --fxaUntrustedOauthApp=https://321done-fxaci.dev.lcip.org --fxaProduction=true --bailAfterFirstFailure=true",
+    "test-functional": "node tests/intern.js",
+    "test-functional-oauth": "node tests/intern.js -- --suites=oauth",
```

Comment:
How come this line has the extra `--` but the others don't? Is it required?

Reply:
Actually, it seems like the extra `--` is wrong.

If I run `npm run test-functional-oauth` with things as they are here, I get the full functional suite. But if I run `node tests/intern --suites=oauth` manually, I get just the OAuth functional suite.

Reply:
good find!